diff --color --unified --recursive --no-dereference mole/examples_C++/Makefile mole-patched/examples_C++/Makefile
--- mole/examples_C++/Makefile	2023-12-06 19:46:20.000000000 -0500
+++ mole-patched/examples_C++/Makefile	2024-02-18 19:43:16.057349689 -0500
@@ -1,12 +1,12 @@
 ifndef ARMA
 # Put here the path to Armadillo
-ARMA = /home/johnny/armadillo-10.2.1
+ARMA = /usr/include
 endif
 
 ifndef EIGEN
 # Put here the path to Eigen (optional)
 # We recommend SuperLU over Eigen for the sparse LU factorization
-#EIGEN = /usr/include/eigen3
+EIGEN = /usr/include/eigen3
 endif
 
 #----------------------------------------------------------------------
diff --color --unified --recursive --no-dereference mole/Makefile mole-patched/Makefile
--- mole/Makefile	2023-12-06 19:46:20.000000000 -0500
+++ mole-patched/Makefile	2024-02-18 19:51:04.895147852 -0500
@@ -1,10 +1,10 @@
 # Put here the path to Armadillo
-ARMA = /home/johnny/armadillo-10.2.1
+ARMA = /usr/include
 
 # Put here the path to Eigen (optional)
 # You won't need it unless you have to solve a linear system in which
 # case we recommend to use SuperLU over Eigen for the sparse LU factorization
-#EIGEN = /usr/include/eigen3
+EIGEN = /usr/include/eigen3
 
 export
 
diff --color --unified --recursive --no-dereference mole/mole_C++/Makefile mole-patched/mole_C++/Makefile
--- mole/mole_C++/Makefile	2023-12-06 19:46:20.000000000 -0500
+++ mole-patched/mole_C++/Makefile	2024-02-18 19:43:33.410688695 -0500
@@ -1,13 +1,13 @@
 ifndef ARMA
 # Put here the path to Armadillo
-ARMA = /home/johnny/armadillo-10.2.1
+ARMA = /usr/include
 endif
 
 ifndef EIGEN
 # Put here the path to Eigen (optional)
 # You won't need it unless you have to solve a linear system in which
 # case we recommend to use SuperLU over Eigen for the sparse LU factorization
-#EIGEN = /usr/include/eigen3
+EIGEN = /usr/include/eigen3
 endif
 
 #----------------------------------------------------------------------
diff --color --unified --recursive --no-dereference mole/tests_C++/Makefile mole-patched/tests_C++/Makefile
--- mole/tests_C++/Makefile	2023-12-06 19:46:20.000000000 -0500
+++ mole-patched/tests_C++/Makefile	2024-02-18 19:43:45.327361356 -0500
@@ -1,6 +1,6 @@
 ifndef ARMA
 # Put here the path to Armadillo
-ARMA = /home/johnny/armadillo-10.2.1
+ARMA = /usr/include
 endif
 #----------------------------------------------------------------------
 MOLE = ../mole_C++
